sdk/eventhub/Microsoft.Azure.EventHubs/tests/Snippets/MigrationGuideSnippets.cs

```diff
@@ -1,418 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-namespace Microsoft.Azure.EventHubs.Tests.Snippets
-{
-    using System;
-    using System.Collections.Generic;
-    using System.Diagnostics;
-    using System.Diagnostics.CodeAnalysis;
-    using System.Linq;
-    using System.Text;
-    using System.Threading;
-    using System.Threading.Tasks;
-    using Microsoft.Azure.EventHubs.Processor;
-    using Microsoft.IdentityModel.Clients.ActiveDirectory;
-    using Xunit;
-
-    /// <summary>
-    ///   The suite of tests defining the snippets used in the migration guide
-    ///   illustrating concepts from Microsoft.Azure.EventHubs and
-    ///   Azure.Messaging.EventHubs in a side-by-side manner.
-    /// </summary>
-    ///
-    [SuppressMessage("Style", "IDE0059:Unnecessary assignment of a value", Justification = "Example assignments needed for snippet output content.")]
-    public class MigrationGuideSnippets
-    {
-        [Fact]
-        [LiveTest]
-        [DisplayTestMethodName]
-        public void CreateWithConnectionString()
-        {
-            #region Snippet:EventHubs_Migrate_T1_CreateWithConnectionString
```

Comment:
@jsquire you did what I did in Key Vault, I see. I think we should consider moving migration source to T2 and take a T1 package dependency instead. Thoughts?